src/features/projects/CreateProjectForm.jsx

```diff
@@ -0,0 +1,156 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { Formik, Form, Field } from 'formik';
+import * as Yup from 'yup';
+import { timeZonesNames } from '@vvo/tzdb';
+import _ from 'lodash';
+
+import { styled } from '../../theme/stitches.config.js';
+import { FormWrapper, FormFieldWrapper, FieldRow, ButtonRow, FormError } from '../../components/Form';
+import Button from '../../components/Button.jsx';
+import SelectField from '../../components/SelectField.jsx';
+import {
+  Toast,
+  ToastTitle,
+  ToastAction,
+  ToastViewport
+} from '../../components/Toast';
+import IconButton from '../../components/IconButton';
+import { Cross2Icon } from '@radix-ui/react-icons';
+import { createProject, selectCreateProjectState, dismissStateMsg } from './projectsSlice.js';
+
+const PageWrapper = styled('div', {
+  maxWidth: '600px',
+  padding: '0 $5',
+  width: '100%',
+  margin: '0 auto'
+});
+
+const Header = styled('div', {
+  fontSize: '24px',
+  fontWeight: '$5',
+  fontFamily: '$roboto',
+  paddingTop: '$8',
+  marginBottom: '$4'
+});
+
+const createProjectSchema = Yup.object().shape({
+  name: Yup.string().required('Enter a project name'),
+  description: Yup.string().required('Enter a short description'),
+  timezone: Yup.string().required('Select a timezone'),
+  availableMLModels: Yup.array().required('Select a ML model'),
+});
+
+const CreateProjectForm = () => {
+  const dispatch = useDispatch();
+  const stateMsg = useSelector(selectCreateProjectState);
+
+  const tzOptions = timeZonesNames.map((tz) => ({ value: tz, label: tz }));
+  const mlModelOptions = [
```

Comment:
We should pull the ML model options from the DB here rather than hard-code them. Just skimming the code here but I think if we use the `fetchModels` [thunk](https://github.com/tnc-ca-geo/animl-frontend/blob/97b29d372733cf0bc2024561652896d3f86fcb49/src/features/projects/projectsSlice.js#L471) and don't pass in any model `_ids`, it should return all of them, and we can build a list of `mlModelOptions` from that.

Reply:
I wasn't able to use the existing thunk because no project is selected on the create-project page. But I added a new thunk and state to store the model options.